src/ForwardShare/getDataParallel.jl

```diff
@@ -47,21 +47,21 @@ function getData(sigma::Vector,pFor::Array{ForwardProbType},Mesh2Mesh::Array=one
 	return Dobs,pFor
 end
 
-function getData(sigma::Vector,pFor::Array{RemoteRef{Channel{Any}}},Mesh2Mesh::Array=ones(length(pFor)),doClear::Bool=false)
-	#= 
+function getData(sigma::Vector,pFor::Array{RemoteChannel},Mesh2Mesh::Array=ones(length(pFor)),doClear::Bool=false)
+	#=
 		load and solve forward problems in parallel
 	=#
 	## Compute Dobs
-	Dobs = Array(RemoteRef{Channel{Any}},length(pFor))
-	
+	Dobs = Array(RemoteChannel,length(pFor))
+
 	# find out which workers are involved
 	workerList = []
 	for k=1:length(pFor)
 		push!(workerList,pFor[k].where)
 	end
 	workerList = unique(workerList)
 	# prepare remoterefs for sigma
-	sigmaRef = Array(RemoteRef{Channel{Any}},maximum(workers()))
+	sigmaRef = Array(RemoteChannel,maximum(workers()))
```

Comment:
Yes. Thanks for the catch. I'll change that. Shows that getDataParallel.jl isn't being tested. I've put adding those tests on my to do list...along with adding tests to JOcTree.